src/AdminConsole/Components/Pages/App/ReportingComponents/TotalCredentialsCountChart.razor

```diff
@@ -0,0 +1,18 @@
+@using Passwordless.Common.Models.Reporting
+<LineChart Id="total-credentials-count-chart" XCategories="@XCategories" Series="@Values"></LineChart>
+
+@code {
+    [CascadingParameter]
```

Comment:
We have to pass the `Task`, as the data isn't initialized yet, we cannot await in our parent component.